man/systemd-sleep.conf.xml

```diff
@@ -77,29 +77,14 @@
       <varlistentry>
         <term>suspend-then-hibernate</term>
 
-        <listitem><para>A low power state where initially user.slice unit is freezed.
```

Comment:
Why is the detailed documentation being removed?

Reply:
Dropped sentences are mostly implementation detail. Not important for users.

Reply:
it's good to have more things clearly documented, it's something we sorely lack in general, so please don't remove this

Reply:
Now it is slightly extended.